group_vars/postgresql/postgresql.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
For the `postgresql` Playbook, the location of this file is fine.  However, when testing this, I found that other Playbooks which use the `pulibrary.postgresql` Role failed to access these or Vault-encrypted variables.

Reply:
Ideally they shouldn't need access to these variables, since they're not in the postgresql group.